test/validation/api/classification/odp_classification_testsuites.h

```diff
@@ -12,16 +12,20 @@
 #include <odp_cunit_common.h>
 #include <stdbool.h>
 
+typedef enum cls_packet_l4_info {
+	CLS_PKT_L4_UDP = 1,
```

Comment:
You should have also CLS_PKT_L4_TCP here so that you could use that name in the code instead of zero. Leaving it out forced you to start the actually defined enums to start from 1 and have the reader guess that value 0 must mean something special.

Reply:
Ack